runner/src/scenario/ceramic/mod.rs

```diff
@@ -23,9 +26,147 @@ pub struct Credentials {
 
 impl Credentials {
     pub async fn from_env() -> Result<Self, anyhow::Error> {
-        let did = DidDocument::new(&std::env::var("DID_KEY").unwrap());
-        let private_key = std::env::var("DID_PRIVATE_KEY").unwrap();
+        let did = DidDocument::new(&std::env::var("DID_KEY").expect("DID_KEY is required"));
+        let private_key = std::env::var("DID_PRIVATE_KEY").expect("DID_PRIVATE_KEY is required");
         let signer = JwkSigner::new(did.clone(), &private_key).await?;
         Ok(Self { signer, did })
     }
+
+    pub async fn admin_from_env() -> Result<Self, anyhow::Error> {
```

Comment:
This should probably be moved to `rust-ceramic`. I had to pull in `ed25519_dalek` directly as I didn't see a public way to do this in the spruceID SSI crates.